seed_services_client/message_sender.py

```diff
@@ -1,29 +1,24 @@
-from demands import JSONServiceClient, HTTPServiceClient
+from .seed_services import SeedServicesApiClient
 
 
-class MessageSenderApiClient(object):
+class MessageSenderApiClient(SeedServicesApiClient):
     """
     Client for Message Sender Service.
 
     :param str auth_token:
-
         An access token.
 
     :param str api_url:
         The full URL of the API.
 
-    """
+    :param JSONServiceClient session:
+        An instance of JSONServiceClient to use
 
-    def __init__(self, auth_token, api_url, session=None, session_http=None):
-        if session is None:
-            session = JSONServiceClient(
-                url=api_url, headers={'Authorization': 'Token ' + auth_token})
+    """
 
-        if session_http is None:
-            session_http = HTTPServiceClient(
-                url=api_url, headers={'Authorization': 'Token ' + auth_token})
-        self.session = session
-        self.session_http = session_http
+    def __init__(self, auth_token, api_url, session=None, **kwargs):
+        super(MessageSenderApiClient, self).__init__(
+            auth_token, api_url, session=session, **kwargs)
```

Comment:
Ah cool, I was under the impression it was required to call the `__init__` in the parent class but less code is better. Removed, thanks!